rocky/reports/report_types/name_server_report/report.py

```diff
@@ -71,9 +71,9 @@ def generate_data(self, input_ooi: str, valid_time: datetime) -> dict[str, Any]:
 
         try:
             ooi = self.octopoes_api_connector.get(Reference.from_str(input_ooi), valid_time)
-        except ObjectNotFoundException as e:
-            logger.error("No data found for OOI '%s' on date %s.", str(e), str(valid_time))
-            raise ObjectNotFoundException(e)
+        except ObjectNotFoundException:
+            logger.error("No data found for OOI '%s' on date %s.", str(ooi), str(valid_time))
```

Comment:
The formatter when using `%s` already implicitly casts/ transforms the argument to a string right?

Reply:
Yes you are right, I just kept that from the original without paying attention to it. I've fixed it now.